src/flash/nor/stm32f1x.c

```diff
@@ -1751,3 +1751,55 @@ const struct flash_driver stm32f1x_flash = {
 	.info = get_stm32x_info,
 	.free_driver_priv = default_flash_free_driver_priv,
 };
+
+/* flash bank gd32vf103 <base> <size> 0 0 <target#>
+ */
+FLASH_BANK_COMMAND_HANDLER(gd32vf103_flash_bank_command)
+{
+	struct stm32x_flash_bank *stm32x_info;
+
+	LOG_WARNING("The gd32vf103 flash target will be removed in June of 2023, please use stm32f1x instead.");
+	/* The reset code are just copy from stm32x_flash_bank_command function */
+	if (CMD_ARGC < 6)
+		return ERROR_COMMAND_SYNTAX_ERROR;
+
+	stm32x_info = malloc(sizeof(struct stm32x_flash_bank));
+
+	bank->driver_priv = stm32x_info;
+	stm32x_info->probed = false;
+	stm32x_info->has_dual_banks = false;
+	stm32x_info->can_load_options = false;
+	stm32x_info->register_base = FLASH_REG_BASE_B0;
+	stm32x_info->user_bank_size = bank->size;
+
+	/* The flash write must be aligned to a halfword boundary */
+	bank->write_start_alignment = bank->write_end_alignment = 2;
+	return ERROR_OK;
+}
+
+static const struct command_registration gd32vf103_command_handlers[] = {
+	{
+		.name = "gd32vf103",
+		.mode = COMMAND_ANY,
+		.help = "gd32vf103 flash command group(reuse stm32f1x)",
```

Comment:
Suggestion for a minor language correction:

```suggestion
		.help = "gd32vf103 flash command group (identical to flash commands from stm32f1x)",
```

Reply:
Fixed.